packages/flutter_test/test/finders_test.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:io';
 import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:path/path.dart';
```

Comment:
looks like this import may have slipped in unintentionally?

Reply:
Yes, but it wasn't me. Argh. VSCode, you're drunk, go home.